README.md

```diff
@@ -184,9 +184,10 @@ $ aws ecs update-service --service sinopia-homepage --region us-west-2 --cluster
 ```
 
 ## Release Management
-The steps to create a tagged release of the Sinopia's Linked Data Editor are as follows:
+The steps to create a major tagged release of the Sinopia's Linked Data Editor are as follows:
 
 1. Update the version in `package.json`
+1. Remove and regenerate `package-lock.json`
```

Comment:
Would `npm i` suffice for this step? Not sure we need to remove the file.

Reply:
Done, updated in latest commit.